core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java

```diff
@@ -574,7 +574,7 @@ public List<TableColumnsMetadata> listTableColumns(Session session, QualifiedTab
 
                 // Collect column metadata from tables
                 metadata.streamTableColumns(connectorSession, tablePrefix)
-                        .forEach(columnsMetadata -> tableColumns.put(columnsMetadata.getTable(), columnsMetadata.getColumns()));
```

Comment:
nit: commit message `may be lazily` -> `may be lazy`

Reply:
thanks, fixed